pkg/interop/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/nspcc-dev/neo-go/pkg/interop
```

Comment:
Won't it conflict with #1865?

Reply:
No as long as micropackages reside in `pkg/interop`. We already have a `rpc/client` <- `examples` <- `pkg/interop` import chain.

Reply:
And will it work fine with neo-go versions? `go get ...@0.98.1`? Looks like it should, but just in case.

Reply:
I have tried it for `neofs-contract` repo, the dependency fetched was neo-go with `@...` version and `pkg/interop` with version from neo-go `go.mod`.
In addition to that contracts fail to built because of an error related to the presence of `replace` directive. Without `replace` changes of `pkg/interop` and main module should be decoupled and done in separate commits.

Reply:
Tested `neofs-contract` here https://github.com/nspcc-dev/neofs-contract/pull/218